OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -259,7 +259,7 @@ public short MovementCostToEnterCell(Actor actor, CPos destNode, Actor ignoreAct
 		}
 
 		// Determines whether the actor is blocked by other Actors
-		public bool CanMoveFreelyInto(Actor actor, CPos cell, Actor ignoreActor, CellConditions check)
+		public bool CanMoveFreelyInto(Actor actor, CPos cell, Actor ignoreActor, CellConditions check, SubCell subCell = SubCell.FullCell)
```

Comment:
We should really group `subCell` with `cell` here too.

This is trickier than the other case because the other args aren't optional, but we can do e38c3be6511b2901ac28b885c936b6c4c1b9ec6d.